### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by Rongjiehuang** [b04901014/FG-transformer-TTS#1]
Hi author, nice implementation! I wonder do we need some pre-trained models for `self.emo_model = MinimalClassifier()`?
Besides, I get audio samples with speaker identity inconsistent with reference audio.

**Comment by b04901014** [comment]
You won't need the pre-trained model for that explicitly. It should load the pretrained wav2vec2 by facebook from huggingface by default.

**Comment by Rongjiehuang** [comment]
```
class MinimalClassifier(pl.LightningModule):
    def __init__(self):
        super().__init__()
        self.wav2vec2 = Wav2vec2Wrapper(pretrain=False)

    def forward(self, x, length=None):
        reps = self.wav2vec2(x, length)
        return reps
```

As you see, the param `pretrain` is set to `False` in default, and I wonder whether it works but not typo?

**Comment by b04901014** [comment]
Sorry for the confusion. I borrowed the module `Wav2vec2Wrapper` from my another project. And that is that parameter is indicating if that module is used for pretraining or fine-tuning instead of whether it is pretrained or not. During pretraining it will output the masks for calculating the pretraining objective, which is not needed.

**Comment by Rongjiehuang** [comment]
Thanks for your clarification.
